st2api/st2api/controllers/v1/packviews.py

```diff
@@ -110,8 +115,10 @@ def get_one(self, name_or_id, *file_path_components):
             # Ignore references to files which don't exist on disk
             raise StackStormDBObjectNotFoundError('File "%s" not found' % (file_path))
 
-        content = self._get_file_content(file_path=normalized_file_path)
-        return content
+        response.headers['Cache-Control'] = 'public, max-age=86400'
+        response.headers['Content-Type'] = mimetypes.guess_type(normalized_file_path)[0]
```

Comment:
It looks like `mimetypes.guess_type` returns `None` if a content type cannot be detected. It's probably safer if we check for that and return `application/octet-stream` or some sane default value?